docs/src/main/tut/README.md

```diff
@@ -113,6 +113,9 @@ libraryDependencies += "io.frees" %% "frees-rpc-dropwizard-client" % "0.11.1"
 
 // optional - for the communication between server and client by using SSL/TLS.
 libraryDependencies += "io.frees" %% "frees-rpc-netty-ssl" % "0.11.1"
+
+// optional - for .proto file generation.
+libraryDependencies += "io.frees" %% "frees-rpc-sbt-idlgen" % "0.11.1"
```

Comment:
For the plugin, I think this would be enough: https://github.com/frees-io/freestyle-rpc/pull/184/files#diff-e9a58680801f40c00e3ed7f80bcd7404R418

Reply:
Ah yes you're right - fixed.